package/CHANGELOG

```diff
@@ -20,6 +20,7 @@ mm/dd/yy richardjgowers, kain88-de, lilyminium, p-j-smith, bdice, joaomcteixeira
   * 0.21.0
 
 Fixes
+  * Create universes under try/except clause.(Issue #2470)(PR #2580)
```

Comment:
Also add your GitHub to the authors list

Reply:
The entries in CHANGELOG are meant to indicate to users what has been fixed, ie here Merging Universes without coordinates.  This entry details what you did to fix it, but it's not useful information here

Reply:
I have updated CHANGELOG. Will take care of it next time.